Day1/PythonIntro.md

```diff
@@ -0,0 +1,37 @@
+| [< Previous (Session Objectives)](Setup.md) | [Day1](../README.md) | [Next (Hello World) >](HelloWorld.md) |
```

Comment:
this is linking to Setup instead of session objectives - both here and in footer.